tests/main/ubuntu-core-config-defaults-once/task.yaml

```diff
@@ -141,3 +142,8 @@ execute: |
         snap get system "service.$service.disable" | MATCH false
     done
     test ! -e /etc/ssh/sshd_not_to_be_run
+
+    # Unmask rsyslog service on core18
+    if is_core18_system; then
+        systemctl unmask rsyslog
```

Comment:
The test does not directly change anything related to rsyslog, however the defaults file contains `rsyslog.disable: true` so this umask seems ok here.